modin/pandas/series.py

```diff
@@ -183,77 +200,175 @@ def _set_name(self, name):
     # should be done to columns of parent.
     _parent_axis = 0
 
+    @_doc_binary_op(operation="addition", bin_op="add")
     def __add__(self, right):
         return self.add(right)
 
+    @_doc_binary_op(operation="addition", bin_op="add", other="left")
     def __radd__(self, left):
         return self.add(left)
 
+    @_doc_binary_op(operation="union", bin_op="and", other="other")
     def __and__(self, other):
         if isinstance(other, (list, np.ndarray, pandas.Series)):
             return self._default_to_pandas(pandas.Series.__and__, other)
         new_self, new_other = self._prepare_inter_op(other)
         return super(Series, new_self).__and__(new_other)
 
+    @_doc_binary_op(operation="union", bin_op="and", other="other")
     def __rand__(self, other):
         if isinstance(other, (list, np.ndarray, pandas.Series)):
             return self._default_to_pandas(pandas.Series.__rand__, other)
         new_self, new_other = self._prepare_inter_op(other)
         return super(Series, new_self).__rand__(new_other)
 
-    def __array__(self, dtype=None):  # noqa: PR01, RT01
+    # add `_inherit_docstrings` decorator to force method link addition.
+    @_inherit_docstrings(pandas.Series.__array__, apilink="pandas.Series.__array__")
```

Comment:
Link is [not added](https://github.com/modin-project/modin/blob/master/modin/utils.py#L122) because method name starts with underscore, so it was added by using of `_inherit_docstrings` exactly on the method.